package/sbp_protocol/src/sbp_router_smoothpose.yml

```diff
@@ -0,0 +1,169 @@
+
```

Comment:
Can we have comments here explaining how this is different from the regular sbp_router.yml?

Reply:
Yes, I will add comments.